cypress.config.js

```diff
@@ -0,0 +1,8 @@
+const { defineConfig } = require('cypress')
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    apiUrl: 'http://localhost:3001'
```

Comment:
Aqui deveria ser algo assim, visto que no Cypress não existe uma propriedade chamada `apiUrl`, assim como existe a `baseUrl`.

```suggestion
    env: {
      apiUrl: 'http://localhost:3001',
    },
```

E então, nos teste que necessitam dessa URL, você obteria tal valor assim: `Cypress.env('apiUrl')`.